R/simplify_fn.R

```diff
@@ -0,0 +1,37 @@
+simple_simplify_predictions_fn <- function(object, predictions) {
+  predictions %>%
+    purrr::reduce(rbind) %>%
+    tibble::as_tibble() %>%
+    tibble::glimpse()
```

Comment:
I think we should leave out the `glimpse()` call here. (This just does some printing to the console, and I think we don't want to implicitly print whenever the user requests predictions -- rather, they should explicitly do that themselves if they want to view the object)

Reply:
Sounds good. I can remove this.